### PR TITLE
Demo tweaks

### PR DESCRIPTION
**Pull request by joetaylor-harness**

### CODE REVIEW

**Comment by joetaylor-harness** [comment]
Added the animated homepage
image

**Comment by joetaylor-harness** [comment]
Added FullscreenLayout to handle WIP component for /signin and /signup
image

**Comment by joetaylor-harness** [comment]
Tweaked studio view with temporary css override to move the controls to the right, flip horizontally and invert. Better matched FooterBar to PixelPoint.
image

**Comment by joetaylor-harness** [comment]
Better matched Repos > Pull Requests list to PP. Colors are not a match yet, but we have a basic method to create colorized badges now.
image

File: packages/canary/src/styles.css [hunk 1]
```diff
@@ -195,3 +195,11 @@
 .entity-list-action-chevron {
   @apply transition-transform transform rotate-0 text-tertiary-background group-hover:text-primary;
 }
+
+/* Temporary studio controls override */
+.react-flow__panel {
```
Comment (abhinavrastogi-harness) [inline packages/canary/src/styles.css:200]: these should not be in canary package, as canary should not be aware of pipeline studio.
we can add them in playground styles.css

**Review by abhinavrastogi-harness — APPROVED** [review]

File: packages/canary/src/styles.css (hunk 1 above)
Comment (joetaylorland) [reply packages/canary/src/styles.css:200]: This was a short term fix for the demo but you are absolutely correct.

**Pull request merged**